examples/python/using_qiskit_level_1.py

```diff
@@ -0,0 +1,146 @@
+"""
+Example showing how to use qiskit (medium)
+
+In QISKit 0.6 we will be working on a pass manager for level 2+ users
+
+Note: if you have only cloned the QISKit repository but not
+used `pip install`, the examples only work from the root directory.
+"""
+
+# Import the QISKit
+import qiskit
+import pprint
+
+# Authenticate for access to remote backends
+# XXX ideally instead of import QConfig we use some localized configuration (windows: registry
+# unix: dotfile, etc)
+
+# registering the backends from the IBM Q Experience
+try:
+    import Qconfig
+    qiskit.register(Qconfig.APItoken)
+except:
+    print("""WARNING: There's no connection with the API for remote backends.
+             Have you initialized a Qconfig.py file with your personal token?
+             For now, there's only access to local simulator backends...""")
+
+local_backends = qiskit.backends.local_backends()
+remote_backends = qiskit.backends.remote_backends()
+
+try:
+    # Create a Quantum and Classical Register and giving a name.
+    qubit_reg = qiskit.QuantumRegister(2, name='q')
+    clbit_reg = qiskit.ClassicalRegister(2, name='c')
+
+    # Making first circuit: bell state
+    qc1 = qiskit.QuantumCircuit(qubit_reg, clbit_reg, name="bell")
+    qc1.h(qubit_reg[0])
+    qc1.cx(qubit_reg[0], qubit_reg[1])
+    qc1.measure(qubit_reg, clbit_reg)
+
+    # Making another circuit: superpositions
+    qc2 = qiskit.QuantumCircuit(qubit_reg, clbit_reg, name="superposition")
+    qc2.h(qubit_reg)
+    qc2.measure(qubit_reg, clbit_reg)
+
+    # Setting up the backend
+    print("(Local Backends)")
+    for backend in local_backends:
+        backend_status = qiskit.backends.status(backend)
+        print(backend, backend_status)
+    my_backend = qiskit.backends.get_backend_instance('local_qasm_simulator')
+    # ideally this should be a filter
+    # my_backend = qiskit.backends.get_backend_instance(filter on local and qasm simulator)
+    print("(Local QASM Simulator configuration) ")
+    pprint.pprint(my_backend.configuration)
+    print("(Local QASM Simulator calibration) ")
+    pprint.pprint(my_backend.calibration)
+    print("(Local QASM Simulator parameters) ")
+    pprint.pprint(my_backend.parameters)
+
+
+    # Compiling the job
+    qobj = qiskit.compile([qc1,qc2])
+    # I think we need to make a qobj into a class
+
+    # Runing the job
+    sim_result = my_backend.run(qiskit.QuantumJob(qobj, preformatted=True))
```

Comment:
Do we really want to expose `QuantumJob`? AFAIK `QuantumJob` is basically a qobj with a timeout and probably another connection related attribute, right? So I was thinking about using something like:
```
sim_result = my_backend.run( qobj, timeout='600')
```
Passing the result of a "compilation" (qobj actually) to a the `run()`method directly, and adding connection related attributes as parameters.

Reply:
In my note i sent that would be a later remove. I agree there is no need for QuantumJob. but i dont want to remove it until we have the qobJ

Reply:
Just to close this, I've been talking with @ajavadia and he showed me that conceptually it's good to have a Job and a Result separately, as someone can ask for specific things of a Job, like where in the queue is it, or what's it state... something that doesn't fit very well with the concept of a Result.

Reply:
Ok, don't want to jump to conclusions for now. Let's postpone this discussion :)